encrypted-media-respec.html

```diff
@@ -2301,6 +2308,29 @@
             <li><p>Resolve <var>promise</var>.</p></li>
           </ol>
         </section>
+        
+        <section id="media-key-session-destroyed">
+          <h4>MediaKeySession destroyed</h4>
+          <p>
+            This algorithm performs steps that are necessary when a <a>MediaKeySession</a> is destroyed without
+            the <a def-id="session-closed-algorithm"></a> algorithm having been run.
+          </p>
+          <p>The following steps are run:</p>
```

Comment:
Add " in parallel[ to the main thread/event loop]".

CDM operations always run in the parallel to the main thread, including the other instance of the `close the <a href="#key-session">session</a>` text in the `close()` algorithm.